app/modules/listings/lib/listings.ts

```diff
@@ -87,8 +102,51 @@ export async function getListingMetadata (listing) {
   }
 }
 
-export async function getListingLongTitle (listing) {
+export async function getElementMetadata (listing, element) {
+  return {
+    title: await getElementTitle(listing, element),
+    url: getElementPathname(listing._id, element._id),
+    smallCardType: true,
+  }
+}
+
+async function getListingLongTitle (listing) {
   const { name, creator } = listing
   const { username } = await app.request('get:user:data', creator)
   return `${name} - ${i18n('list_created_by', { username })}`
 }
+
+async function getElementTitle (listing, element) {
+  const { entity } = element
+  const { name, creator } = listing
+  const { username } = await app.request('get:user:data', creator)
+  return `${entity.title} - ${name} - ${i18n('list_created_by', { username })}`
+}
+
+export async function askUserConfirmationAndRemove (removeElementPromise, deletingData) {
+  if (deletingData) {
+    app.execute('ask:confirmation', {
+      // i18n suggestion:  "delete_element_confirmation": "Are you sure you want to delete this element. Choosing yes will delete the element and the following data:  \"**%{deletingData}**\"?",
+      confirmationText: i18n('delete_element_confirmation', { deletingData }),
```

Comment:
Underscore keys aren't needed anymore to do string interpolation
```suggestion
      confirmationText: i18n('Are you sure you want to **delete this element**? That will also delete your comment: %{comment}…', { comment: comment.slice(0, 20) }),
```